refinery/data_set_manager/views.py

```diff
@@ -998,11 +998,35 @@ def put(self, request, uuid, format=None):
             new_rank = request.data.get('rank', None)
 
             if id:
-                attribute_order = AttributeOrder.objects.get(
-                    assay__uuid=uuid, id=id)
+                try:
+                    attribute_order = AttributeOrder.objects.get(
```

Comment:
Can you go ahead and fix the (). https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#python